src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/BreakEndVariantType.java

```diff
@@ -120,4 +137,77 @@ private static String getIDString(final NovelAdjacencyReferenceLocations narl, f
                     (forUpstreamLoc ? "1" : "2") ;
         }
     }
+
+    /**
+     * Generic variant type for suspected "translocations", including what could be a breakpoint for a duplication or
+     * an inter-chromosome novel adjacency.
+     */
+    public static final class TransLocBND extends BreakEndVariantType {
+        private static Map<String, String> emptyMap = Collections.emptyMap();
+
+        private TransLocBND(final NovelAdjacencyReferenceLocations narl, final boolean forUpstreamLoc,
+                            final ReferenceMultiSource reference, final SAMSequenceDictionary referenceDictionary,
+                            final boolean basesFirst, final boolean bracketPointsLeft) {
+            super(getIDString(narl, forUpstreamLoc), emptyMap,
+                    extractBasesForAltAllele(narl, forUpstreamLoc, reference, referenceDictionary), bracketPointsLeft,
+                    getMateRefLoc(narl, forUpstreamLoc), basesFirst, forUpstreamLoc);
+        }
+
+        public static Tuple2<BreakEndVariantType, BreakEndVariantType> getOrderedMates(final NovelAdjacencyReferenceLocations narl,
```

Comment:
There is a documentation for that [here](https://github.com/broadinstitute/dsde-methods-sv/blob/1d8789605a542be0db65cb2666d0d2b1b4223fc4/docs/ideas/transLoc/transLocationVcfProposal.md) in our private repo. 
Admittedly it is accessible only for us. But I think that is OK for now. 

I've been refraining from adding tests for these new features through this whole complex SV PR series (see #3431) because of the nightmarish memory back in the days of adding tests for ins/del cases&mdash;getting the locations exactly right takes as much as, if not more, energy to get these logic right.
I've been keeping track of the test coverage in this prototyping package for a while, and yes, it is [VERY VERY BAD](https://codecov.io/gh/broadinstitute/gatk/tree/a23536ab54b723e35afb1f8012b5548d74664872/src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/prototype) right now. What I have in mind, is to have a pipeline in place, generate call set for evaluation (correctness, format, etc), and add tests based on feedback from at least one round of call set review. Rightnow, I'm the only one that is running this parallel program (not even a pipeline yet like `StructuralVariationDiscoveryPipelineSpark`). And before test coverage get above 75%, I have no intention of hooking up this path and replace the master pipeline.